File: tools/bench/answers/go_sonnet/s2/02_vowels.go
package main

import "fmt"

func main() {
	s := "the quick brown fox jumps over the lazy dog"
	count := 0
	for _, ch := range s {
		switch ch {
		case 'a', 'e', 'i', 'o', 'u':
			count++
		}
	}
	fmt.Println(count)
}

File: tools/bench/answers/rust_haiku/s1/02_vowels.rs
fn main() {
    let s = "the quick brown fox jumps over the lazy dog";
    let mut count = 0;

    for ch in s.chars() {
        match ch {
            'a' | 'e' | 'i' | 'o' | 'u' => count += 1,
            _ => {}
        }
    }

    println!("{}", count);
}
